python_modules/dagster/dagster/_cli/workspace/cli_target.py

```diff
@@ -133,13 +133,37 @@ def get_workspace_load_target(kwargs: Mapping[str, str]):
             "grpc_port",
             "grpc_socket",
         )
+        python_files = kwargs["python_file"]
+
         working_directory = get_working_directory_from_kwargs(kwargs)
-        return PythonFileTarget(
-            python_file=check.str_elem(kwargs, "python_file"),
-            attribute=check.opt_str_elem(kwargs, "attribute"),
-            working_directory=working_directory,
-            location_name=None,
-        )
+
+        if len(python_files) == 1:
+            return PythonFileTarget(
+                python_file=python_files[0],
+                attribute=check.opt_str_elem(kwargs, "attribute"),
+                working_directory=working_directory,
+                location_name=None,
+            )
+        else:
+            # multiple files
+
+            if kwargs.get("attribute"):
+                raise UsageError(
+                    "If you are specifying multiple files you cannot specify an attribute."
+                )
+
+            return CompositeTarget(
```

Comment:
You could alternately return a list of targets from this function vs. a new target type that wraps a list

Reply:
That would a non-trivial refactor, involving making WorkspaceProcessContext accept a list of targets, unless I am missing something